experiments/pagination/README.md

```diff
@@ -0,0 +1,98 @@
+# Use of Paging for large sets
+
+This experiment demonstrates why pagination might provide a better experience, and allows hands-on, immediate feedback
+on pagination fields.
+
+## Problem
+At some point, the optimal query has been created, using all the performance enhancements, but still produces a lot
+of hits and takes a long time to complete. Such a delay can be a bad experience for the user and uses a lot of 
+system resources. 
+
+## Goals
+This experiment tries to show the power of Pagination and the use of [Querybuilder](https://helpx.adobe.com/experience-manager/6-5/sites/developing/using/reference-materials/javadoc/com/day/cq/search/QueryBuilder.html)
+properties.  A customer’s search may include introduction of their own custom APIs, and not have the UI call
```

Comment:
@top Ditch the "include" here.  Suggst: "A customizer's search implementation may introduce their own custom APIs, effectively bypassing the QueryBuilder API..."